internal/dag/dag.go

```diff
@@ -156,20 +156,45 @@ func (hc *HeaderMatchCondition) String() string {
 const (
 	// QueryParamMatchTypeExact matches a querystring parameter value exactly.
 	QueryParamMatchTypeExact = "exact"
+
+	// QueryParamMatchTypePrefix matches a querystring parameter value is
+	// prefixed by a given string.
+	QueryParamMatchTypePrefix = "prefix"
+
+	// QueryParamMatchTypeSuffix matches a querystring parameter value is
+	// suffixed by a given string.
+	QueryParamMatchTypeSuffix = "suffix"
+
+	// QueryParamMatchTypeRegex matches a querystring parameter value against
+	// given regular expression.
+	QueryParamMatchTypeRegex = "regex"
+
+	// QueryParamMatchTypeContains matches a querystring parameter value
+	// contains the given string.
+	QueryParamMatchTypeContains = "contains"
+
+	// QueryParamMatchTypeIgnoreCase enables case insensitive matching of the
+	// querystring parameter value.
+	QueryParamMatchTypeIgnoreCase = "ignorecase"
```

Comment:
I think we can drop this one since it has its own field in the QueryParamMatchCondition, correct me if I'm missing something though

Reply:
Yep, seems I forgot to remove this from an earlier stage.